### DISCUSSION TITLE
3/23(목) 클로저

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#27] [General]
오늘은 클로저에 대한 개념을 공부해 볼 것 입니다. 익명 함수이기 때문에 내가 언제 사용했었나? 하겠지만 주로 $0 또는 in 으로 많이 봤습니다. 클로저 또한 개념자체가 생소해서 어려울 수 있습니다. 하다가 어려우면 좀 많이 혼자 했는데 어렵다 그럼 낑낑 대지 말고 한번 물어보면 코드 예시 들어주겠습니다. 오늘도 열심히하자~ 🦁

**Comment by Eunho0922** [comment]
넵 오늘도 최대한 열심히 해보겠습니다!!
